Makefile

```diff
@@ -387,20 +407,50 @@ assets_nocore: \
 	ebpf_nocore
 	$(CMD_GO) run github.com/shuLhan/go-bindata/cmd/go-bindata $(IGNORE_LESS52) -pkg assets -o "assets/ebpf_probe.go" $(wildcard ./user/bytecode/*.o)
 
+
+$(TARGET_LIBPCAP):
+	cd lib/libpcap && \
```

Comment:
Can we first determine if the git submodule has been loaded?

Most people's usage habits are to directly clone the git repository without loading the submodule. It is recommended to check here whether the submodule has been loaded, and if not, automatically execute the command `git submodule update --init`.

Reply:
Regarding the warning information about the `ld` command, you can refer to https://github.com/google/gopacket/issues/734. 

However, it has little impact and can be temporarily ignored.

Reply:
Added `test -f ./lib/libpcap/configure || git submodule update --init ./lib/libpcap`.